apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -341,6 +342,16 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		return this.find(query, options || {});
 	}
 
+	findOpenByVisitorIds(visitorIds: string[], options?: FindOptions<ISubscription>): FindCursor<ISubscription> {
+		const query = {
+			't': 'l' as const,
```

Comment:
For this case, i think would be better to have an idx on `open + v._id` as rooms having `v._id` are for sure type `t`. So the index may not be exclusive enough.

wanna give it a try?